packages/salesforcedx-vscode-core/test/vscode-integration/commands/forceProjectCreate.test.ts

```diff
@@ -122,14 +122,18 @@ describe('Force Project Create', () => {
       inputBoxStub.returns(PROJECT_NAME);
       const response = await gatherer.gather();
       expect(response.type).to.equal('CONTINUE');
-      expect((response as ContinueResponse<ProjectName>).data.projectName).to.equal(PROJECT_NAME);
```

Comment:
nit: we could add a typecheck to remove the need for the as casting here. 

```const isContinueResponse = (
  response: CancelResponse | ContinueResponse<ProjectName>
): response is ContinueResponse<ProjectName> => {
  return (response as ContinueResponse<ProjectName>).data !== undefined;
};```

Then the test would look like 
```
      expect(isContinueResponse(response)).to.equal(true);
      if (isContinueResponse(response)) {
        expect(response.type).to.equal('CONTINUE');
        expect(response.data.projectName).to.equal(PROJECT_NAME);
      }
      ```
      Check out type predicates here: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates

Reply:
@klewis-sfdc did you decide against the typecheck here?

Reply:
Nope, meant to ask you about this just to clarify further - Ill take a look at this this afternoon

Reply:
updated!